mm2src/coins/lightning.rs

```diff
@@ -1098,4 +1099,10 @@ impl MmCoin for LightningCoin {
 
     // Todo: This uses default data for now for the sake of swap P.O.C., this should be implemented probably when implementing order matching if it's needed
     fn is_coin_protocol_supported(&self, _info: &Option<Vec<u8>>) -> bool { true }
+
+    fn on_disabled(&self) -> Result<(), AbortedError> {
+        AbortableSystem::abort_all(&self.platform.coin.as_ref().abortable_system)
```

Comment:
I don't think so, as that will abort all futures related to the platform coin and not the coin we want..

Reply:
`platform.abortable_system` is a subsystem of the platform coin https://github.com/KomodoPlatform/atomicDEX-API/blob/b2f1eb112ec8834ef7c637d59a2e03f18672b4cb/mm2src/coins/lightning/ln_platform.rs#L183
https://github.com/KomodoPlatform/atomicDEX-API/blob/b2f1eb112ec8834ef7c637d59a2e03f18672b4cb/mm2src/coins/lightning/ln_platform.rs#L198
While `platform.coin` is the platform coin https://github.com/KomodoPlatform/atomicDEX-API/blob/b2f1eb112ec8834ef7c637d59a2e03f18672b4cb/mm2src/coins/lightning/ln_platform.rs#L154 so `AbortableSystem::abort_all(&self.platform.coin.as_ref().abortable_system)` aborts the platform coin related futures.

Reply:
Yes, correctly. I agree with @shamardy

Reply:
I guess it confuses because I put the Lightning's `abortable_system` into `Platform` structure. It was just more convenient because `Platform` is accessible from all places where we need to spawn lightning's futures

Reply:
ok I was confused too. thanks